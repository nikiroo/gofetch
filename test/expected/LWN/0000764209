         [$] STRENGTHENING USER-SPACE SPECTRE V2 PROTECTION        

  [Kernel] Sep 5, 2018 21:47 UTC (Wed) (corbet)

  o Reference: 0000764209
  o News link: https://lwn.net/Articles/764209
  o Source link: 


    [$]   Sorry,   this  article  is  currently  available  to  LWN
    suscribers only [https://lwn.net/subscribe/].



