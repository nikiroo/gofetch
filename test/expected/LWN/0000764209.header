0[$] Strengthening user-space Spectre v2 protection	null/LWN/0000764209		70
i  [Kernel] Sep 5, 2018 21:47 UTC (Wed) (corbet)
i
i    The  Spectre  variant  2  vulnerability  allows the speculative
i    execution   of  incorrect  (in  an  attacker-controllable  way)
i    indirect  branch  predictions,  resulting  in  the  ability  to
i    exfiltrate  information  via side channels. The kernel has been
i    reasonably  well  protected  against this variant since shortly
i    after  its  disclosure in January. It is, however, possible for
i    user-space  processes  to  use Spectre v2 to attack each other;
i    thus  far,  the  mainline  kernel has offered relatively little
i    protection  against  such  attacks.  A  recent [1]proposal from
i    Jiri  Kosina  may  change  that  situation, but there are still
i    some disagreements around the details.
i    
i    
i    
i    [1] https://lwn.net/ml/linux-kernel/nycvar.YFH.7.76.1809041619-
i    510.15880@cbobk.fhfr.pm/
i
