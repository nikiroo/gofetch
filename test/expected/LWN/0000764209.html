<!DOCTYPE html>
<html>
<head>
  <meta http-equiv='content-type' content='text/html; charset=utf-8'>
  <meta name='viewport' content='width=device-width, initial-scale=1.0'>
  <style type='text/css'>
    body { margin: 1em 15%; }
  </style>
</head>
<body>
<div class='story'>
	<h2>[$] Strengthening user-space Spectre v2 protection</h2>
	<div class='details'>([Kernel] Sep 5, 2018 21:47 UTC (Wed) (corbet))</div>
	<br/>
    <ul>
    <li>Reference: <a href=''>0000764209</a></li>
        <li>News link: <a href='https://lwn.net/Articles/764209'>https://lwn.net/Articles/764209</a></li>
        <li>Source link: <a href=''></a></li>
    </ul>
	<br/>
	<div class='content' style='text-align: justify'>
		[$] Sorry, this article is currently available to LWN suscribers only [https://lwn.net/subscribe/].
	</div>
<hr/>
</div>
</body>
